core/compiler.cpp

```diff
@@ -434,6 +451,9 @@ torch::jit::Module CompileGraph(const torch::jit::Module& mod, CompileSpec cfg)
           (!(cfg.lower_info.forced_fallback_modules.size() == 0 &&
              cfg.partition_info.forced_fallback_operators.size() == 0 && isBlockConvertible) ||
            outputIsCollection)) {
+        if (!static_params.empty()) {
+          RewriteInputsWithParams(g, params);
```

Comment:
Should this be a lowering pass?

Reply:
It could be a lowering pass. However, I'm not so sure about what happens with conversion stage if the input are stored in params. Any thought on this? @narendasan

Reply:
cool let me change it to a lowering pass.